qiskit_nature/second_q/hamiltonians/quadratic_hamiltonian.py

```diff
@@ -32,7 +31,7 @@ def _is_antisymmetric(mat: np.ndarray, rtol: float = 1e-5, atol: float = 1e-8) -
     return np.allclose(mat, -mat.T, rtol=rtol, atol=atol)
 
 
-class QuadraticHamiltonian(Hamiltonian, TolerancesMixin):
+class QuadraticHamiltonian(PolynomialTensor, Hamiltonian):
```

Comment:
I'm not sure I understand. Are you suggesting to set the class attributes `QuadraticHamiltonian.atol` and `QuadraticHamiltonian.rtol` to the values passed to `__init__`? I don't think that should be done since that would affect other instances too.

Reply:
Yes I had misremembered myself, that `rtol` and `atol` are actually class attributes as dictated by the `TolerancesMixin`. In that case, please document that the `rtol` and `atol` keywords of `__init__` are only used during `__init__` and discarded afterwards

Reply:
Done.